OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -45,7 +45,7 @@ public class AssetBrowserLogic : ChromeLogic
 		int currentFrame;
 
 		[ObjectCreator.UseCtor]
-		public AssetBrowserLogic(Widget widget, Action onExit, World world)
+		public AssetBrowserLogic(Widget widget, Action onExit, World world, Dictionary<string, MiniYaml> logicArgs)
```

Comment:
What if someone wants to pass a dictionary as one of the logic class' parameters?
Don't forget that the code that handles the initializations is very generic and handles the initialization of *all* widget logic classes.